thirdparty/scripts/build_credis.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -x
+
+# Cause the script to exit if a single command fails.
+set -e
+
+TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)/../
+ROOT_DIR=$TP_DIR/..
+
+if [[ "${RAY_USE_NEW_GCS}" = "on" ]]; then
+  pushd "$TP_DIR"
+    rm -rf credis
+    git clone --recursive https://github.com/ray-project/credis
+  popd
+
+  pushd "$TP_DIR/credis"
+    git checkout 6be4a739ab5e795c98402b27c2e254f86e3524ea
+
+    # TODO(pcm): Get the build environment for tmalloc set up and compile redis
+    # with tmalloc.
+    # NOTE(zongheng): if we don't specify MALLOC=jemalloc, then build behiavors
+    # differ between Mac (libc) and Linux (jemalloc)... This breaks our CMake
+    # rules.
+    pushd redis && make -j MALLOC=jemalloc && popd
+    pushd glog && cmake -DWITH_GFLAGS=off . && make -j && popd
+    # NOTE(zongheng): DO NOT USE -j parallel build for leveldb as it's incorrect!
+    pushd leveldb && CXXFLAGS="$CXXFLAGS -fPIC" make && popd
+
+    mkdir build
+    pushd build
+      cmake ..
+      make -j
```

Comment:
either do a parallel make here or just omit the `-j`?

Reply:
-j uses the maximum degree of parallelism